packages/flutter_tools/templates/list_detail_app/analysis_options.yaml.tmpl

```diff
@@ -0,0 +1,16 @@
+# Static analysis allows you to find problems before executing a single line of code.
+# You may customize the static analysis by following this guide:
+# https://dart.dev/guides/language/analysis-options
+include: package:pedantic/analysis_options.yaml
+
+analyzer:
+  strong-mode:
+    implicit-casts: false
+    implicit-dynamic: false
+
+# You may enable or disable various lint rules. A full list of rules
+# can be found at https://dart-lang.github.io/linter/lints/
+linter:
+  rules:
+    prefer_void_to_null: true
+    always_specify_types: false
```

Comment:
we shouldn't have an analysis_options file here. it should just default to the current default.